src/main/twirl/gitbucket/core/main.scala.html

```diff
@@ -1,4 +1,4 @@
-@(title: String, repository: Option[gitbucket.core.service.RepositoryService.RepositoryInfo] = None)(body: Html)(implicit context: gitbucket.core.controller.Context)
+@(title: String, repository: Option[gitbucket.core.service.RepositoryService.RepositoryInfo] = None, showHeaderContents: Boolean = true)(body: Html)(implicit context: gitbucket.core.controller.Context)
```

Comment:
At first I tried to hide header content (search bar, plugins, ...) by checking user is logged-in and allowAnonymousAccess setting but that causes a case that anonymous user can not see header content on every pages even if allowAnonymousAccess is true.
So I introduced `showHeaderContents` flag to `main.scala.html`.